iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergAcidUtil.java

```diff
@@ -204,7 +247,7 @@ public static long parseFilePosition(Record rec) {
     return rec.get(FILE_READ_META_COLS.get(MetadataColumns.ROW_POSITION), Long.class);
   }
 
-  public static long computeHash(StructProjection struct) {
+  public static long computeHash(StructLike struct) {
```

Comment:
why is this change?

Reply:
Partition objects retrieved during Merge task are in StructLike object.